lib/types/operation.js

```diff
@@ -90,7 +90,7 @@ var Operation = module.exports = function (parent, scheme, operationId, httpMeth
     if (innerType && innerType.toString().toLowerCase() === 'boolean') {
       param.allowableValues = {};
       param.isList = true;
-      param['enum'] = ['true', 'false'];
+      param['enum'] = [true, false]; // use actual primitives
```

Comment:
No, I'm just saying that technically, boolean in 1.X should have been boolean values and not strings, but for the sake of robustness there's probably no harm supporting both.